docs/INSTALLING.md

```diff
@@ -19,9 +19,22 @@ The directory structure will look like the following:
 ```shell
 code-corps-ember/    # → Root folder for this project
 ├── app/
-├── config/
-├── dist/
-├── ...              # → More standard Ember.js files
+    ├── abilities/   # → Used by `ember-can` to manage user access
+    ├── components/  # → Handles UI behavior (i.e. MVC controller)
+    ├── controllers/ # → Specialized components used by routes
```

Comment:
The comments? Controllers are specialized components, are they not? At least that's what ember documentation says.